client/src/visualiser-src/graph-visualiser/util/util.ts

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable */
```

Comment:
This file is a bit ugly, but easy to tidy up. Skipped for now to avoid investing too much time into an approach that might not be what we go with.